openpype/hosts/fusion/plugins/publish/render_local.py

```diff
@@ -23,42 +21,28 @@ def process(self, instance):
         # This plug-in runs only once and thus assumes all instances
         # currently will render the same frame range
         context = instance.context
-        key = "__hasRun{}".format(self.__class__.__name__)
-        if context.data.get(key, False):
-            return
-        else:
-            context.data[key] = True
+        self.render_once(context)
 
-        current_comp = context.data["currentComp"]
         frame_start = context.data["frameStartHandle"]
         frame_end = context.data["frameEndHandle"]
         path = instance.data["path"]
         output_dir = instance.data["outputDir"]
 
-        ext = os.path.splitext(os.path.basename(path))[-1]
-
-        self.log.info("Starting render")
-        self.log.info("Start frame: {}".format(frame_start))
-        self.log.info("End frame: {}".format(frame_end))
-
-        with comp_lock_and_undo_chunk(current_comp):
-            result = current_comp.Render({
-                "Start": frame_start,
-                "End": frame_end,
-                "Wait": True
-            })
-
-        if "representations" not in instance.data:
-            instance.data["representations"] = []
-
-        collected_frames = os.listdir(output_dir)
+        basename = os.path.basename(path)
+        head, ext = os.path.splitext(basename)
+        files = [
+            f"{head}{str(frame).zfill(4)}{ext}" for frame in range(frame_start, frame_end + 1)
```

Comment:
I don't know how to make that string shorter

Reply:
```suggestion
            f"{head}{str(frame).zfill(4)}{ext}"
            for frame in range(frame_start, frame_end + 1)
```

Reply:
Question is if there should be used hardcoded padding of `4`. If anyone would use frame start `10001` it would not work.

Reply:
Currently the creator creates a saver node with the path `renderCompositingMain..exr`
Fusion will check before before the `.exr` if there is any numbers or `#`. If not it will add the frame number with a minimum of 4 numbers.
So as long as the creator doesn't change how the path looks like that zfill(4) will work what ever number you'll put in